frontend/src/components/PostCard.js

```diff
@@ -17,6 +17,8 @@ const PostCard = ({ post, link, isComment = false, children, ...others }) => {
 
   let auth = useSelector((state) => state.auth);
   let authUser = auth.user;
+  let authUserId = authUser.id + '';
```

Comment:
This is not ideal although this is fastest.
`let a = '' + 281e-26 // '2.81e-24'` Not precise
Use num.toString() or String(num) instead.

Reply:
ok. I will make a update for it.